docs/templates/Project Context/Coordinate Reference System in WKT/README.md

```diff
@@ -0,0 +1,20 @@
+Coordinate Reference System in WKT
+==================================
+
+```
+concept {
+    IfcContext:RepresentationContexts -> IfcGeometricRepresentationContext
+    IfcContext:Phase -> IfcLabel_1
+    IfcContext:ObjectType -> IfcLabel_2
+    IfcContext:LongName -> IfcLabel_3
+    IfcGeometricRepresentationContext:HasCoordinateOperation -> IfcCoordinateOperation:SourceCRS
+    IfcCoordinateOperation:TargetCRS -> IfcCoordinateReferenceSystem
+    IfcCoordinateReferenceSystem:WellKnownText -> IfcWellKnownText:CoordinateReferenceSystem
+    IfcCoordinateReferenceSystem:Name -> IfcLabel_0
```

Comment:
I believe we should bind `Name` to be equal to `WKT` as it stands in the documentation.

Reply:
done